src/Symfony/Component/Mailer/Bridge/Mailtrap/Transport/MailtrapApiTransport.php

```diff
@@ -49,9 +50,19 @@ public function __toString(): string
         return \sprintf('mailtrap+api://%s', $this->getEndpoint());
     }
 
+    /**
+     * @return $this
+     */
+    public function setInboxId(mixed $inboxId): static
+    {
+        $this->inboxId = (int) $inboxId;
```

Comment:
```suggestion
    public function setInboxId(int $inboxId): static
    {
        $this->inboxId = $inboxId;
```

The cast will happen automatically when calling `setInboxId()`.

Reply:
```suggestion
    public function setInboxId(?int $inboxId): static
    {
        $this->inboxId = $inboxId;
```

Actually, since we're always calling setInboxId() this needs to accept null.